website/docs/reference/resource-configs/tags.md

```diff
@@ -164,10 +198,51 @@ seeds:
 
 </File>
 
+### Apply tags to saved queries
+
+<VersionBlock lastVersion="1.8">
+
+Applying tags to saved queries is only available in dbt Core versions 1.9 and later.
+
+</VersionBlock>
+
+<VersionBlock firstVersion="1.9">
+
+This following example shows how to apply a tag to a saved query in the `dbt_project.yml` file. The saved query is then tagged with `order_metrics`.
+
+<File name='dbt_project.yml'>
+
+```yml
+[saved-queries](/docs/build/saved-queries):
+  jaffle_shop:
+    customer_order_metrics:
+      +tags: order_metrics
+```
+
+</File>
+
+The second example shows how to apply multiple tags to a saved query in the `semantic_model.yml` file. The saved query is then tagged with `order_metrics` and `hourly`.
+
+<File name='semantic_model.yml'>
+
+```yaml
+saved_queries:
+  - name: test_saved_query
+    description: "{{ doc('saved_query_description') }}"
+    label: Test saved query
+    config:
+      tags: 
+        - order_metrics
+        - hourly
```

Comment:
Putting the tags inside the config is not how I built this to work or expected it to work, but it DOES work, so that's pretty cool, and I think it's more in line with how other things work, so... thanks for making this better I guess! lol